vllm/core/scheduler.py

```diff
@@ -1051,6 +1056,17 @@ def _preempt(
                 preemption_mode = PreemptionMode.RECOMPUTE
             else:
                 preemption_mode = PreemptionMode.SWAP
+
+        if self.num_cumulative_preemption % 500 == 0:
```

Comment:
I think this should be logged every time we have a preemption.

Preemptions happening means that we are in a bad state.

Reply:
hmm I feel like that's too spammy and metrics & stats logger can already cover it. Lmk if this is a hard blocker, but imo printing once in a a while is enough for most cases